dev/com.ibm.ws.webcontainer/src/com/ibm/ws/webcontainer/webapp/WebAppRequestDispatcher.java

```diff
@@ -1414,8 +1414,12 @@ public void dispatch(ServletRequest request, ServletResponse response, final Dis
                                                     CollaboratorHelper.allCollabEnum);
         } catch (ServletErrorReport ser) {
             // PK79464
-            if (dispatcherRethrowSER&&httpServletReq.getAttribute(WebContainerConstants.IGNORE_DISPATCH_STATE) == null)
+            if (dispatcherRethrowSER&&httpServletReq.getAttribute(WebContainerConstants.IGNORE_DISPATCH_STATE) == null) {
+	        if (WCCustomProperties.DONOTCLOSEOUTPUTONFORWARDEX 
+                    || com.ibm.ws.webcontainer.osgi.WebContainer.getServletContainerSpecLevel() >= 60) 
```

Comment:
I don't want us to change behavior for Servlet 6.0 at this point in the game. We can discuss this in the next servlet feature if we want to change the behavior by default.

Reply:
spec level test removed